lib/plugins/aws/package/compile/events/websockets/lib/api.js

```diff
@@ -26,6 +48,7 @@ module.exports = {
           Description:
             this.serverless.service.provider.websocketsDescription || 'Serverless Websockets',
           ProtocolType: 'WEBSOCKET',
+          Tags: tags,
```

Comment:
One minor nitpick - let's add it to the CF resource definition only conditionally - right now we will assign an emtpy object which will result in a bit different CF template which will cause a redeploy even for people that do not have any tags - let's avoid that.

Reply:
hey @tjurdzinski - would you be able to address that minor change?

Reply:
@pgrzesik yes, I will do it in 1-2 days.

Reply:
thanks 🙇

Reply:
@pgrzesik I did changes. If tags are empty I don't add 'Tags' to cloudformation template.

Reply:
Thanks, well done @tjurdzinski 👏